src/org/labkey/test/tests/ProjectCreatorUserTest.java

```diff
@@ -197,8 +201,35 @@ public void testCreateProjectByTemplateWithSubfolderAndPermission() throws Comma
 
     private String createProject(Map<String, Object> params) throws IOException
     {
-        PostCommand<CommandResponse> command = new PostCommand<>("admin", "createProject");
-        command.setParameters(params);
+        // CreateProjectAction is not a real API action, so we POST form data not JSON
```

Comment:
Is this intended only for this test, or might it also be useful for tests to generally use when creating projects?

Reply:
Good call @labkey-chrisj ... yes, I thought about moving this to the Java API because it could be useful. Unfortunately, current implementation requires modern Java (streams and all that), but the Java API still supports Java 8, so I didn't take the leap right now. We are talking about loosening the requirements on the Java API, but that won't happen immediately.